gandalf.js

```diff
@@ -2,30 +2,31 @@
  * Extras Plugin
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.